examples/grpc/grpc-proto.arrai

```diff
@@ -34,6 +34,7 @@ let endpoints = app.endpoints where !//.str.has_prefix(.@item.name, "enum ");
                 enum ${//.rex.sub(.name, `^\S*\s`, "")} {
                     ${.stmts >> $`${.action};`::\i}
                 }`:::\n}
+            ${endpoints >> (proto.wrapSequence(.)).grpcType::\i}
```

Comment:
This should work now:
```suggestion
            ${endpoints >> proto.wrapSequence(.).grpcType::\i}
```

Reply:
ok, works with the latest version of arrai.